pommerman/__init__.py

```diff
@@ -41,4 +41,5 @@ def make(config_id, agent_list, game_state_file=None):
 
     env.set_agents(agent_list)
     env.set_init_game_state(game_state_file)
+    env.render(mode=render_mode)
```

Comment:
this will actually call render. you should do set_render(mode=render_mode) and that will set a self._render_mode var on the env side.